docs/tutorial/progress-bar.md

```diff
@@ -1,35 +1,56 @@
 # Progress Bar in Taskbar (Windows, macOS, Unity)
 
-On Windows a taskbar button can be used to display a progress bar. This enables
-a window to provide progress information to the user without the user having to
-switch to the window itself.
+## Overview
 
-On macOS the progress bar will be displayed as a part of the dock icon.
+On Windows, a taskbar button can be used to display a progress bar. This enables
+a window to provide progress information to the user without the need of switching to
+the window itself.
+
+![Taskbar Progress Bar][taskbar-progress-image]
+
+On macOS, the progress bar will be displayed as a part of the dock icon.
 
 The Unity DE also has a similar feature that allows you to specify the progress
```

Comment:
Would be helpful to expand "Unity DE" to users who are unfamiliar with Linux-specific terms.
```suggestion
On Linux, the Unity graphical interface also has a similar feature that allows you to specify the progress
```